cucumber/src/test/resources/requirements/TSH Sample KB.feature

```diff
@@ -0,0 +1,22 @@
+Feature: The user should be able to see all test dates and results for a case
+
+  @single
+  Scenario: Case with two episodes
+    Given the configured case Case4 is stored on the server
+    And I start the client application
+    When I see the case Case4 as the current case
+    Then I should see these episode dates:
```

Comment:
maybe nicer to put the expected attribute names, values and reference range in a table